lib/functions.php

```diff
@@ -1781,3 +1781,103 @@ function jirafeau_add_ending_slash($path)
 {
     return $path . ((substr($path, -1) == '/') ? '' : '/');
 }
+
+/**
+ * Function allowing the sending of mail from a form
+ * see more : https://www.php.net/manual/fr/function.mail.php
+ * @param $transmitter: the transmitter of the email
+ * @param $recipient: the recipient of the email
+ * @param $message is the message of the email
+ * @return true if the mail send.
+ */
+function jirafeau_send_mail($transmitter, $recipient, $message, $link, $email_subject, $filename, $expireDate, $password)
+{
+    $message = '<html>
+    <style>
+        #h1 {
+            color:#663d1c;
+        }
+        #body {
+            justify-content:center;
+            display: flex;
+        }
+        #strong {
+            color: #663d1c;
+        }
+        #expireFile {
+            font-size: 14px;
+            margin-top: 0px;
+        }
+        .h3 {
+            margin: 0px;
+        }
+        .block {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin-top:20px;
+            margin-bottom:20px;
+        }
+        #text {
+            margin: 30px;
+        }
+        #button {
+            text-transform: uppercase;
+            outline: 2px;
+            background: #663d1c;
+            width: 35%;
+            padding: 15px;
+            color: #FFFFFF;
+            font-size: 14px;
+            cursor: pointer;
+            border-radius: 20px;
+        }
+        #footer {
+            font-size: 12px;
+            display: flex;
+            flex-direction:column;
+            align-items: center;
+        }
+        #mail {
+            background-color:#f4f4f4;
+            max-width: 60%;
+            display:flex;
+            flex-direction: column;
+            align-items:center;
+        }
+    </style>
```

Comment:
And why do you add id everywhere (#h1 on h1, #body on body). I think that it's very not necessary. Just use css selector body, h1, ... without the #